industrial_ci/src/docker.sh

```diff
@@ -177,6 +199,7 @@ function ici_build_default_docker_image() {
 
 function ici_generate_default_dockerfile() {
   cat <<EOF
+$inject_qemu
```

Comment:
Just wondering what this does. Also why is this in lower-case?
Since this line is not conditional to QEMU so all build is affected, it'd be nice to clarify.

Reply:
> Just wondering what this does. 

Nothing (anymore), it is just a left-over from my tests with build stages, I will remove it.

> Also why is this in lower-case?

At first it was an internal variable (=lower case..)